ntrip_client:
  namespace: "/ntrip"
  node_name: "ntrip_client"
  debug: False
  host: "sapos.geonord-od.de"
  port : "2101"
  mountpoint: "RTCM4G"
  ntrip_version: 'None'
  authenticate: True
  username: "gast"
  password: "gast"
  ssl: False
  cert: 'None'
  key: 'None'
  ca_cert: 'None'
  rtcm_message_package: 'rtcm_msgs'